client/components/layout/NavBar.vue

```diff
@@ -40,8 +40,16 @@
       <hr/>
       <section>
       <header><h2>
-        <NavLink to="/account" name="My Account" :activatable="false"/>
+        <NavLink to="/account" name="Account Settings" :activatable="false"/>
       </h2></header>
+      <ul>
+        <li>
+          <NavLink to="/account" name="Shared Account"/>
+        </li>
+        <li>
+          <NavLink to="/user" :name="$store.getters.username || 'Log in'"/>
```

Comment:
Feeling iffy about:
- "Shared Account" vs. "@username". I kinda like "Login Settings" better? Not sure
- "Shared Account" vs. "Log in"; or in general, having the header present but not really "clickable" feels a bit odd, but again not sure what the best remedy is. Also, things on the "Log in" page are all phrased in terms of the shared account, not the user, so it's a little weird that it's replacing the user link. 

[nit] "Log In" capitalization?

Reply:
yeah I had "Login Settings" originally and was on the fence about that. I'll switch it back if you think it's better.